docs/development.md

```diff
@@ -64,10 +58,7 @@ kubectl apply -f examples/facebook-opt-125m/finetuned-model.yaml
 Cleanup.
 
 ```sh
-docker run -it \
-  -e PROJECT=$(gcloud config get project) \
-  -e TOKEN=$(gcloud auth print-access-token) \
-  substratus-installer gcp-down.sh
+make gcp_installer_down
```

Comment:
Perhaps `make uninstall CLOUD=gcp`

Reply:
Also pushing this one off until we have another platform we support. I'm all for it at that point!